h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -3570,9 +3570,9 @@ def compare_frames_local_onecolumn_NA(f1, f2, prob=0.5, tol=1e-6, returnResult=F
     temp2 = f2.as_data_frame(use_pandas=False)
     assert (f1.nrow==f2.nrow) and (f1.ncol==f2.ncol), "The two frames are of different sizes."
     if oneLessRow:
-        lastRow = f2.nrow
```

Comment:
this is weird, all I can see is a whitespace change

Reply:
I accidentally pushed to rel-yau.  I thought I reverted it back but guessed not.

Reply:
I re-did the whole thing and tried again here.  Please check again.  It should contain three files to be fixed.  Good catch.  Thank you Honza.